010-AzureDevOps/Coach/Solution-06.md

```diff
@@ -4,17 +4,4 @@
 
 ## Notes & Guidance
 
-This is the only section you need to include.
-
-Use general non-bulleted text for the beginning of a solution area for this challenge
-
-- Then move into bullets
-  - And sub-bullets and even
-    - sub-sub-bullets
-
-Break things apart with more than one bullet list
-
-- Like this
-- One
-- Right
-- Here
+-the application and the docker image are deployed to Azure DevOps, it's not ready. should have to update to 3.0.0 .net core at least, inside project file. 
```

Comment:
Can you clarify what you mean here?  In challenge 5 the app is built and deployed to ACR.  These recommendations afterwards sound like they should be mentioned during the build process of the app in challenge 5

Reply:
already added to 5.